app/models/miq_server.rb

```diff
@@ -152,6 +152,7 @@ def start
     #############################################################
     # Start all the configured workers
     #############################################################
+    clean_heartbeat_files
```

Comment:
Is this something we want to do conditionally?  By that, I mean only worry about this if we are using a file based heartbeat?  This can be done within the method of course, just thought I would bring it up.